appium-tests/common/common.spec.js

```diff
@@ -228,11 +228,18 @@ describe('Contacts Android', function () {
     function checkSession(done) {
         if (failedToStart) {
             fail('Failed to start a session');
-            done();
         }
+        done();
```

Comment:
@alsorokin if I don't put it there, for successful test runs, the `beforeEach` times out with something like `Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL`.